ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -4,13 +4,17 @@
 import com.fasterxml.jackson.databind.node.NullNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.Lists;
```

Comment:
Nit: this isn't in alphabetical order with the other imports.

Reply:
I still haven't gotten my IntelliJ settings back to what I want after laptop refresh. I used to be able to rely on it to do the right thing automatically.